dhis-2/dhis-services/dhis-service-validation/src/main/java/org/hisp/dhis/outlierdetection/service/DefaultOutlierDetectionService.java

```diff
@@ -212,4 +216,28 @@ private OutlierDetectionMetadata getMetadata( OutlierDetectionRequest request, L
         metadata.setMaxResults( request.getMaxResults() );
         return metadata;
     }
+
+    /**
+     * Returns outlier values using the algorithm defined in the request.
+     *
+     * @param request the {@link OutlierDetectionRequest}.
+     * @return a {@link OutlierDetectionMetadata} instance.
```

Comment:
hm... looks like a mismatch of docs and actual type to me. If there isn't anything to add I'd think `@param` and `@return` can be left out on a private method.

Reply:
Correct, good spot, fixed now.